sympy/stats/joint_rv.py

```diff
@@ -224,7 +231,11 @@ def _sample_pymc3(cls, dist, size):
 
         with pymc3.Model():
             pymc3_rv_map[dist.__class__.__name__](dist)
-            return pymc3.sample(size, chains=1, progressbar=False)[:]['X']
+            samples = pymc3.sample(size, chains=1, progressbar=False, random_seed=seed)[:]['X']
```

Comment:
Is this size querying related to adding the seed in pymc3? It looks like it fixes a different problem. If so, it might be better to change this as quick hotfix of a different PR.

Reply:
`size` is reflective of the size of each `sample`. The argument related to `seed` is `random_seed`.